src/components/Newsletter/NewsletterForm/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React, {FC} from 'react';
+import './index.scss';
+
+interface FormProps {
+  buttonType?: 'primary' | 'secondary';
+}
+
+const NewsletterForm: FC<FormProps> = ({buttonType = 'primary'}) => {
+  return (
+    <form
+      className="newsletter-form"
+      onSubmit={(e) => {
+        // TODO - Logic for saving the email for the upcoming newsletter
```

Comment:
Can you complete the flow for it? Write the API calling fucntion. So, before merging I'll just replace the URL.